deploy/kubernetes/gke/bookkeeper.statefulset.yml

```diff
@@ -39,8 +39,6 @@ data:
     BK_ledgerDirectories: "/bookkeeper/data/ledgers"
     BK_indexDirectories: "/bookkeeper/data/ledgers"
     BK_zkServers: zookeeper
-    # the default manager is flat, which is not good for supporting large number of ledgers
-    BK_ledgerManagerType: "hierarchical"
```

Comment:
We could leave this line so that it is explicit this important configuration parameter

Reply:
`ledgerMangerType` is already deprecated long time ago. it doesn't make any sense to leave it here. we used the wrong parameter here so I don't see a reason to have it though. for people who is new to bookkeeper, he should be knowing less settings as possible.